src/DataGrid.tsx

```diff
@@ -2,32 +2,53 @@ import React, {
   forwardRef,
   useState,
   useRef,
-  useLayoutEffect,
   useMemo,
+  useLayoutEffect,
+  useEffect,
+  useImperativeHandle,
+  useCallback,
   createElement
 } from 'react';
 
+import EventBus from './EventBus';
+import InteractionMasks from './masks/InteractionMasks';
 import HeaderRow from './HeaderRow';
 import FilterRow from './FilterRow';
-import Canvas, { CanvasHandle as DataGridHandle } from './Canvas';
+import RowRenderer from './RowRenderer';
+import SummaryRow from './SummaryRow';
 import { ValueFormatter } from './formatters';
-import { getColumnMetrics, getHorizontalRangeToRender, getViewportColumns, getScrollbarSize } from './utils';
-import { CellNavigationMode, SortDirection } from './common/enums';
+import {
+  assertIsValidKey,
+  getColumnMetrics,
+  getColumnScrollPosition,
+  getHorizontalRangeToRender,
+  getScrollbarSize,
+  getVerticalRangeToRender,
+  getViewportColumns
+} from './utils';
+
 import {
   CalculatedColumn,
   CheckCellIsEditableEvent,
   Column,
-  RowsUpdateEvent,
+  Filters,
+  FormatterProps,
   Position,
   RowExpandToggleEvent,
-  SelectedRange,
   RowRendererProps,
+  RowsUpdateEvent,
   ScrollPosition,
-  Filters,
-  FormatterProps
+  SelectedRange,
+  SelectRowEvent
 } from './common/types';
+import { CellNavigationMode, SortDirection } from './common/enums';
 
-export { DataGridHandle };
+export interface DataGridHandle {
+  scrollToColumn: (colIdx: number) => void;
+  scrollToRow: (rowIdx: number) => void;
+  selectCell: (position: Position, openEditor?: boolean) => void;
+  openCellEditor: (rowIdx: number, colIdx: number) => void;
```

Comment:
May be we should remove `openCellEditor` or at least change the params to match `selectCell`. Probably best suited for a separate PR

Reply:
Will remove it in a future PR